src/java.base/share/classes/java/lang/System.java

```diff
@@ -797,6 +797,15 @@ public static native void arraycopy(Object src,  int  srcPos,
      *     <td>The module name of the initial/main module</td></tr>
      * <tr><th scope="row">{@systemProperty jdk.module.main.class}</th>
      *     <td>The main class name of the initial module</td></tr>
+     * <tr><th scope="row">{@systemProperty file.encoding}</th>
+     *     <td>The name of the default charset. Users may specify
+     *     {@code UTF-8} or {@code COMPAT} on the command line to the value.
```

Comment:
The wording could imply that only those two values can be supplied.
It could be rephrased to say that *if* the property is supplied on the command line
it overrides the default UTF-8.

Reply:
That was intentional. Only those two are supported, others continue to work as before (but not supported).

Reply:
Still it leaves an uncomfortable feeling, perhaps remedied by an "other values have unspecified behavior"
or the "other values are implementation specific".

Reply:
Added the clarifying sentence to the spec.